README.md

```diff
@@ -1 +1,55 @@
-# product-catalog
\ No newline at end of file
+# Product Catalog
+In this project, we have created a console app that will help to keep a record of different types of things like: books and music albums. The data is be stored in JSON files.
+
+## Built With
+
+- Ruby
+- Gem
+- Terminal
+- PostgresSQL
+- RSpec
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- go to the directory that you want to clone the repository in.
+- right click and choose Git Bash here.
+- type "git clone https://github.com/Mutalenic/product-catalog.git".
+- go to the repository folder in your command prompt.
+- run bundle install.
+
+## Video Presentation
+[Video Link](https://drive.google.com/file/d/1ZaMa-XHFxW86GoPU3w58fPHEOQit6DEc/view?usp=sharing)
+
```

Comment:
- [ ] Great video presentation, but as it is a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md) for this project, under `Project documentation`,  " `You should also highlight some interesting piece of code or something you built that you are very proud of.` ", please kindly highlight a piece of code that you find interesting or is proud of making.